binder/views.py

```diff
@@ -635,18 +635,14 @@ def _get_with_ids(self, pks, request, with_map, where_map):
 		Agg = GroupConcat if connections[self.model.objects.db].vendor == 'mysql' else OrderableArrayAgg
 
 		for field in with_map:
-			next = self._follow_related(field)[0].model
-			view_class = self.router.model_view(next)
+			vr = self.virtual_relations.get(field, None)
+
+			next = self._follow_related(field)[0]
```

Comment:
Any reason why this was changed? I don't particularly disagree with it, but I don't see a clear reason why. Maybe we should rename the `next` variable to something more describing to. `next_relation` or something?

Reply:
Ah, that's the result of some reordering I did for clarity. vr is used all over in this "for" loop, so it made more sense to me to start by declaring it at the top.

I'd be fine with renaming it, but I consider that a separate refactor (which the reordering would also be, strictly speaking)

Reply:
I was referring to the next, which first was a model, and now is a relation?

Reply:
Sure, we can rename it